docs/getting-started.md

```
@@ -0,0 +1,100 @@
+
+**PostgreSQL**
+- **Local Instance**: Set up and run a PostgreSQL instance locally.
+
+**PGAdmin**
+- **Local Instance**: Run a pre-configured instance of PgAdmin that connects to your local PostgreSQL database.
+
+**API Gateway**
+- **Local Instance**: Launch the ITM API-gateway in your local environment.
+
+**AppSwitcher**
+- **Repository**:https://github.com/it-at-m/appswitcher-server
```

Markdownlint (0.37.0)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)